tests/src/test/scala/common/WhiskProperties.java

```diff
@@ -349,6 +349,10 @@ public static boolean onLinux() {
         return osname.equalsIgnoreCase("linux");
     }
 
+    public static int getMaxActionSizeMB(){
+        return Integer.parseInt(getProperty("whisk.action.size.max", "10"));
```

Comment:
@chetanmeh, where is `whisk.action.size.max` set?

Reply:
Currently its not set and hence defaults to 10 MB. I wish to configure this (via system property) when we run system basic test against out setup with the max value that the cluster supports.